ocs_ci/ocs/must_gather/const_must_gather.py

```diff
@@ -0,0 +1,564 @@
+    'ceph_mgr_module_ls_--format_json-pretty',
+    'ceph_mgr_services_--format_json-pretty',
+    'ceph_mon_dump_--format_json-pretty',
+    'ceph_mon_stat_--format_json-pretty',
+    'ceph_osd_blacklist_ls_--format_json-pretty',
+    'ceph_osd_blocked-by_--format_json-pretty',
+    'ceph_osd_crush_class_ls_--format_json-pretty',
+    'ceph_osd_crush_dump_--format_json-pretty',
+    'ceph_osd_crush_rule_dump_--format_json-pretty',
+    'ceph_osd_crush_rule_ls_--format_json-pretty',
+    'ceph_osd_crush_show-tunables_--format_json-pretty',
+    'ceph_osd_crush_weight-set_dump_--format_json-pretty',
+    'ceph_osd_crush_weight-set_ls_--format_json-pretty',
+    'ceph_osd_df_--format_json-pretty',
+    'ceph_osd_df_tree_--format_json-pretty',
+    'ceph_osd_dump_--format_json-pretty',
+    'ceph_osd_getmaxosd_--format_json-pretty',
+    'ceph_osd_lspools_--format_json-pretty',
+    'ceph_osd_numa-status_--format_json-pretty',
+    'ceph_osd_perf_--format_json-pretty',
+    'ceph_osd_pool_ls_detail_--format_json-pretty',
+    'ceph_osd_stat_--format_json-pretty',
+    'ceph_osd_tree_--format_json-pretty',
+    'ceph_osd_utilization_--format_json-pretty',
+    'ceph_pg_dump_--format_json-pretty',
+    'ceph_pg_stat_--format_json-pretty',
+    'ceph_progress_--format_json-pretty',
+    'ceph_progress_json',
+    'ceph_progress_json_--format_json-pretty',
+    'ceph_quorum_status_--format_json-pretty',
+    'ceph_report_--format_json-pretty',
+    'ceph_service_dump_--format_json-pretty',
+    'ceph_status_--format_json-pretty',
+    'ceph_time-sync-status_--format_json-pretty',
+    'ceph_versions_--format_json-pretty',
+]
+
+GATHER_COMMANDS_OTHERS = [
+    'buildconfigs.yaml', 'builds.yaml', 'configmaps.yaml', 'cronjobs.yaml',
+    'daemonsets.yaml', 'db-noobaa-db-0.yaml', 'deploymentconfigs.yaml',
+    'deployments.yaml', 'endpoints.yaml', 'events.yaml',
+    'horizontalpodautoscalers.yaml', 'imagestreams.yaml', 'jobs.yaml',
+    'my-alertmanager-claim-alertmanager-main-0.yaml',
+    'my-alertmanager-claim-alertmanager-main-1.yaml',
+    'my-alertmanager-claim-alertmanager-main-2.yaml',
+    'my-prometheus-claim-prometheus-k8s-0.yaml',
+    'my-prometheus-claim-prometheus-k8s-1.yaml',
+    'noobaa-core-0.log', 'noobaa-core-0.yaml',
+    'noobaa-db-0.log', 'noobaa-db-0.yaml', 'noobaa-default-backing-store.yaml',
+    'noobaa-default-bucket-class.yaml', 'noobaa.yaml',
+    'ocs-storagecluster-cephblockpool.yaml', 'ocs-storagecluster-cephcluster.yaml',
+    'ocs-storagecluster-cephfilesystem.yaml', 'openshift-storage.yaml',
+    'persistentvolumeclaims.yaml', 'pods.yaml', 'pools_rbd_ocs-storagecluster-cephblockpool',
+    'previous.log', 'previous.log', 'registry-cephfs-rwx-pvc.yaml', 'replicasets.yaml',
+    'replicationcontrollers.yaml', 'rook-ceph-mon-a.yaml', 'rook-ceph-mon-b.yaml',
+    'rook-ceph-mon-c.yaml', 'routes.yaml', 'secrets.yaml', 'services.yaml',
+    'statefulsets.yaml', 'storagecluster.yaml',
+]
+
+GATHER_COMMANDS_CEPH_4_5 = [
```

Comment:
for 4.5 ceph commands https://github.com/openshift/ocs-operator/blob/de48c9c00f8964f0f8813d7b3ddd25f7bc318449/must-gather/collection-scripts/gather_ceph_resources#L33-L87

https://github.com/openshift/ocs-operator/blob/de48c9c00f8964f0f8813d7b3ddd25f7bc318449/must-gather/collection-scripts/gather_ceph_resources#L91-L92

and rbd ls -p <pool name>